test/rsrc/fish/simple.fish

```diff
@@ -0,0 +1,71 @@
+
+# An 'argparse' description for global options for 'beet'.
+function __fish_beet_global_optspec
```

Comment:
I was meant to add a comment to ask to move this to a `.fish` file but forgot. Happy to see that it was done nevertheless!